lib/util.js

```diff
@@ -499,7 +499,7 @@ exports.construct_hostname = function (region, account)
  */
 exports.isPutCommand = function (sqlText)
 {
-  return (sqlText.substring(0, 3).toUpperCase() === "PUT");
+  return (sqlText.replace(/\s+/g, '').substring(0, 3).toUpperCase() === "PUT");
```

Comment:
Do we need to replace white space from whole SQL or just to trim before and after the command? Better to use trim() kind of function.